### PR TITLE
Add support of parentheses

### PR DESCRIPTION
**Pull request by Manki233**
Replace Quotes and Backslash with the Parentheses.

It's should be compatible Quotes and Backslash.

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by heinrich5991** [comment]
Hi! Thanks for your first pull request.

Can you describe what the goal of your PR is?

**Comment by Manki233** [comment]
> Hi! Thanks for your first pull request.
> 
> Can you describe what the goal of your PR is?

I wan't use parentheses replace the quotes and backslash. here is a sample :

`bind space "+jump;\"+fire\"" -> bind space (+jump;(+fire))`

It's good work on my computer.

**Comment by Manki233** [comment]
I'm try to compile it with my Ubuntu 20.04 server.

**Comment by def-** [comment]
I don't think it fits our syntax. What's wrong with quotes and backslashes?

**Comment by Manki233** [comment]
> I don't think it fits our syntax. What's wrong with quotes and backslashes?

Sorry, I think it's get more nature of syntax. Of course someone don't think it.

**Comment by Bamcane** [comment]
I think `bind("mouse1", "+fire;+toggle cl_dummy_hammer 1 0")` is even better

**Comment by Manki233** [comment]
> I think is even better`bind("mouse1", "+fire;+toggle cl_dummy_hammer 1 0")`

If we wan't to nesting the bind, like: `bind q (bind mouse1 (+toggle cl_dummy_hammer 1 0))`

It will be like this: `bind ("q", "bind (\"mouse1\", \"+toggle cl_dummy_hammer 1 0\")")`

emmm.... nesting quotes is annoying for me. I really don't feel this like bask.

**Comment by Manki233** [comment]
> > I don't think it fits our syntax. What's wrong with quotes and backslashes?
> 
> Sorry, I think it's get more nature of syntax. Of course someone don't think it.



> 1D6A9904203603CBCA7FED95B9AA81AD
> 
> So, I get some warning on my server. I don't have warning on my Ninja windows compiler. may It's error on other compiler. I try to get some websearch, but I still don't know how to solve it.

I solved it problem now. _may?_

File: .github/workflows/main.yml
Comment (Bamcane) [inline .github/workflows/main.yml:1]: What's this?

**Comment by Manki233** [comment]
i tink this a bad idea. i closed it.

**Pull request closed without merging**